### ISSUE TITLE
Install script no longer installs on Mac OS X Catalina 10.15

### ISSUE THREAD

**Issue by bval** [warpling/Macmoji#52]
The install script exits without errors but it does not install working text replacements. This is because the key `NSUserDictionaryReplacementItems` is no longer where OS X stores text replacements. They have been moved into iCloud as a dictionary under `com.apple.TextInput`.

To make Macmoji CLI installable again we will either need to find a new mechanism to feed them into the appropriate place or pivot to AppleScript to import them into the PrefPane.

**Comment by bval** [comment]
This is likely the same issue as reported under #39.

**Comment by warpling** [comment]
Hey @bval, super sorry for never getting back to you. I spent a day trying to get the old bash script working again but would like to migrate it to AppleScript or Python. How do you imagine an Apple Script being able to import them into PrefPane? I'm not terribly familiar. 

Also where is `com.apple.TextInput`? I don't see it under `~/Library/Preferences/`

**Comment by bval** [comment]
> Hey @bval, super sorry for never getting back to you.

Oh wow, I totally forgot about this issue. 😂 No worries at all.

> I spent a day trying to get the old bash script working again but would like to migrate it to AppleScript or Python. How do you imagine an Apple Script being able to import them into PrefPane? I'm not terribly familiar.

Not an AppleScript developer either, but I believe it's possible to open the PrefPane, and then click and fill in fields using their Accessibility IDs. That would definitely be an implementation of last resort. It's been a while since I investigated other mechanisms to do this.
 
> Also where is `com.apple.TextInput`? I don't see it under `~/Library/Preferences/`

Because it's in iCloud it's now in `~/Library/Mobile\ Documents/com~apple~TextInput`.

Hope this helps.

**Comment by warpling** [comment]
After a lot of messing with this I finally resolved to remove the install script. 😕 I spent a day or so mucking around and couldn't figure it out (admittedly this is really outside my wheelhouse!). I think the plist drag/drop method still works perfectly for most. 

I would be more than happy to add a script via PR if you or @camlloyd have ideas for creating one (perhaps in Apple Script?) but for now want to remove it and close these issues to avoid confusion with users on newer OSes. Let me know what you think!
